### PR TITLE
Switch voice recognition backend to Sphinx

### PR DESCRIPTION
**Pull request by Maneren**
Implements the Sphinx voice recognition backend proposed by gavlna.

### CODE REVIEW

**Comment by Maneren** [comment]
We want to squash merge this, so please just review and I'll do the merging.

File: spot/audio/main.py [hunk 1]
```diff
@@ -31,17 +41,21 @@ def run(self, stopper: Stop, callback: Callable[[str], None]) -> None:
...
                 self.stop_function(True)
                 return
 
             try:
-                text = recognizer.recognize_vosk(audio).lower()
-            except sr.UnknownValueError:
-                return
+                text = recognizer.recognize_sphinx(
+                    audio,
+                    language="cs-CZ",
```
Comment (vokymir) [inline spot/audio/main.py]: Language CZ why? Commands are in English, does this help to account for our accent, or?

**Review by vokymir — COMMENTED** [review]
LGTM, except for the langue cs-CZ

**Comment by Maneren** [comment]
Whoops, fixed that.

**Review by vokymir — APPROVED** [review]
dobro

**Review by Bonifac0 — APPROVED** [review]
lgtm

**Pull request merged**